extensions/google-chrome/src/components/ChromeListItems.tsx

```diff
@@ -9,6 +9,27 @@ export class ChromeListItems {
   public static TabHistory = HistoryItem;
 }
 
+// Helper function to safely get favicon for potentially invalid URLs
+function getSafeFavicon(url: string) {
+  // Filter out known problematic URL schemes
+  const invalidSchemes = ["javascript:", "data:", "about:", "chrome:", "file:"];
+  const urlLower = url.toLowerCase().trim();
+
+  // Check if URL starts with any invalid scheme
+  if (invalidSchemes.some((scheme) => urlLower.startsWith(scheme))) {
+    return { source: "" };
+  }
```

Comment:
I like the validation. The validation's error case needs better handling though. Silently discarding invalid user input can be confusing and frustrating. This should show an error message describing the problem and what to do about it.